db/seeds.rb

```diff
@@ -1,17 +1,33 @@
 # frozen_string_literal: true
 
-Rake::Task['fetch:references'].invoke # fetch YOKSIS references
-Rake::Task['import:all'].invoke # import common static data
+config = ActiveRecord::Base.configurations[Rails.env]
 
-# Produced data for beta environment
-if Rails.env.beta? || Rails.env.development?
-  Dir[Rails.root.join('db', 'beta_seed', '*.rb')].sort.each do |seed|
-    load seed
-  end
+if ENV['SYNC'].eql?('true')
+  Rake::Task['fetch:references'].invoke
+  Rake::Task['import:all'].invoke
+else
+  connection = "PGPASSWORD=#{config['password']} psql \
+               -h #{config['host']} \
+               -d #{config['database']} \
+               -U #{config['username']} "
+
+  `tar xvzf #{Rails.root.join('db', 'encrypted_data', 'static_data.tar.gz').to_s} -C tmp/`
```

Comment:
Unutulan `git lfs pull`'ları için dostça bir önlem alsak mı?  Önce dosyanın varlığını kontrol edip uyarıcı bir mesajla çıkmak gibi.